src/ACEtk/interpretation/EPRData.hpp

```diff
@@ -0,0 +1,27 @@
+class EPRData {
+  Table table;
```

Comment:
You own a table. The <functional> header was added for the use of `reference_wrapper`, no?